docs/providers/gemini.md

```diff
@@ -7,7 +7,3 @@
     'url' => env('GEMINI_URL', 'https://generativelanguage.googleapis.com/v1beta/models'),
 ],
 ```
-
-## Limitations
-
-- The structured output is not supported.
```

Comment:
Please mention the structure mode is currently supported using JSON mode. The Model Structured output is still not supported.